colibre/scripts/bh_merger_mass_fractions.py

```diff
@@ -0,0 +1,96 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy.stats as stats
+
+from swiftsimio import load
+from velociraptor.autoplotter.objects import VelociraptorLine
+from velociraptor.tools.lines import binned_median_line
+from velociraptor.autoplotter.plot import scatter_x_against_y
+
+from unyt import mh, cm, Gyr
+
+# Set the limits of the figure.
+mass_bounds = [1e5, 3e10]  # Msun
+value_bounds = [0, 1.02]  # dimensionless
+bins = 25
+plot_scatter = True
+
+
+def get_data(filename):
+
+    data = load(filename)
+
+    masses = data.black_holes.subgrid_masses.to("Msun")
+    accreted_masses = data.black_holes.total_accreted_masses.to("Msun")
+    values = 1.0 - accreted_masses / masses
+    values = values * unyt.dimensionless
```

Comment:
This script will crash on line 26 because you haven't imported `unyt`

Reply:
Ah, yes, thanks for catching that. This is the only script I couldn't properly test out since I was using older snapshots from an  older version of the jets branch, that had an error in the units of `data.black_holes.total_accreted_masses`.